src/com/itacademy/eshop/product/Toy.java

```diff
@@ -0,0 +1,39 @@
+package com.itacademy.eshop.product;
+
+import com.itacademy.eshop.interfaces.Available;
+import com.itacademy.eshop.product.types.Category;
+
+public abstract class Toy extends Product implements Available {
+        private boolean isPlushy;
+    private boolean isAvailable;
+        public Toy (boolean isPlushy, String name, double price, Category category) {
+            super(name, price, category);
+            this.isPlushy = isPlushy;
+        }
+
+        public boolean getIsPlushy(){
+            return isPlushy;
+        }
+        public boolean getIsAvailable(){
+            return isAvailable;
+        }
+        public void setIsPlushy(boolean newIsPlushy){
+            isPlushy = newIsPlushy;
+        }
+        @Override
+        public boolean updateAvailability(){
```

Comment:
name implies that availability of object will be updated (changed) and with current implementation you just get that availability, change either name or implementation